src/tests/unit/onbase-dao.test.js

```diff
@@ -1,33 +1,79 @@
-const chai = require('chai');
-const chaiAsPromised = require('chai-as-promised');
-const proxyquire = require('proxyquire');
-const sinon = require('sinon');
-
-const conn = require('../../api/v1/db/oracledb/connection');
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import config from 'config';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+import createError from 'http-errors';
+
+// config must be replaced before connection can be imported
+sinon.replace(config, 'get', () => ({ oracledb: {} }));
+const conn = require('api/v1/db/oracledb/connection');
```

Comment:
I think this can be put in the place while proxyquiring the DAO: https://github.com/osu-mist/students-api/blob/master/src/tests/unit/students-dao.test.js#L19-L32

Reply:
The config error occurs when you require `connection.js`. I tried it there, it must come first.